pom.xml

```diff
@@ -107,7 +107,7 @@
     <pulsar.spark.version>${pulsar.spark.scala12.version}</pulsar.spark.version>
     <pulsar.spark.scala12.version>3.1.1.4</pulsar.spark.scala12.version>
     <pulsar.spark.scala13.version>3.4.1.1</pulsar.spark.scala13.version>
-    <confluent.version>5.5.0</confluent.version>
+    <confluent.version>6.2.15</confluent.version>
```

Comment:
Could we upgrade to `7.3.x` which has been running in production? Also check if we can take this forward: #5758